app/views/collections/_form.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= simple_form_for @form, html: { class: 'editor' } do |f| %>
+  <div id="descriptions_display">
+
+  <div class="well">
```

Comment:
No problem.  When I rebase my collections metadata PR I may move it down to the bottom.  My collections PR removes the additional fields button so that there is only a main form.